packages/keystone/preprocessors/schemaDoc.js

```diff
@@ -60,9 +61,11 @@ const { GQL_CUSTOM_SCHEMA_TYPE } = require('../schema')
  * @return {string}
  * TODO: localize error message
  */
-const formatError = (errorConfig) => (
-    '`' + JSON.stringify(errorConfig, null, 2) + '`'
-)
+const formatError = (errorConfig) => {
+    const error = new GQLError(errorConfig, {})  // render error with DEFAULT_LOCALE
+    const errorFields = pick(error.extensions, ['code', 'type', 'message', 'messageForUser', 'correctExample'])
```

Comment:
Maybe add a "variable" field?

Reply:
You really don't want to look to another places if you want to understand how this function relly works. I can but really it's decrease readability. In this cases it's better to hardcode.

Reply:
I don't think so. At the moment don't really well support this field. May be in a future.